src/graph/TraverseExecutor.cpp

```diff
@@ -142,7 +142,7 @@ nebula::cpp2::SupportedType TraverseExecutor::calculateExprType(Expression* exp)
         case Expression::kInputProp: {
             auto* propExp = static_cast<const AliasPropertyExpression*>(exp);
             const auto* propName = propExp->prop();
-            if (inputs_ == nullptr) {
+            if (inputs_ == nullptr || !inputs_->hasData()) {
```

Comment:
if inputs_ == nullptr , Is it will leads to crash by  `inputs_->hasData()` ?

Reply:
I think won't because the short circuit ||